vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold.cc

```diff
@@ -209,6 +210,10 @@ void Uphold::DisconnectWallet() {
     return;
   }
 
+  ledger_->database()->SaveEventLog(
+      ledger::log::kWalletDisconnected,
+      static_cast<std::string>(ledger::kWalletUphold) + "/" + wallet->address);
```

Comment:
nit: could probably also just do `std::string(ledger::kWalletUphold)` if preferred.

Reply:
`static_cast<std::string>` is approach that is defined in chromium style guides